### PR TITLE
!cra added

### PR DESCRIPTION
**Pull request by barrybtw**
image

solves #404

### CODE REVIEW

File: src/features/commands.ts [hunk 1]
```diff
@@ -1196,6 +1196,27 @@ Authentication is a critical part of most web applications. Here are some resour
       });
     },
   },
+  {
+    words: ["!cra", "!create-react-app"],
+    help: "Provides a list of popular backend- and meta-frameworks",
```
Comment (kristersd) [inline src/features/commands.ts:1201]: Please adjust the help tag

**Comment by kristersd** [comment]
Let's include link to the official react documentation to [start a new project without framework](https://react.dev/learn/start-a-new-react-project#can-i-use-react-without-a-framework).
As well adjust the links to the guide themselves instead of the homepage.

File: src/features/commands.ts (hunk 1 above)
Comment (vcarl) [reply src/features/commands.ts:1201]: Whatcha thinking @kristersd? like: ?
```suggestion
    help: "Provides alternatives to Create React App, which is no longer recommended.",
```

File: src/features/commands.ts [hunk 2]
```diff
@@ -1196,6 +1196,27 @@ Authentication is a critical part of most web applications. Here are some resour
...
+    handleMessage: (msg) => {
+      msg.channel.send({
+        embeds: [
+          {
+            title: "create-react-app is deprecated",
+            type: EmbedType.Rich,
+            description: `
+create-react-app is deprecated and no longer recommended for use. It is not maintained and has a large number of security vulnerabilities. Please use Vite or Next.js instead.
+Vite: https://vitejs.dev/
+Next.js: https://nextjs.org/
```
Comment (vcarl) [inline src/features/commands.ts:1212]: What about:
```suggestion
create-react-app is deprecated and no longer recommended for use. It is not maintained and has a large number of security vulnerabilities. Please use [Vite](https://vitejs.dev/) or [Next.js](https://nextjs.org/) instead. The [React docs](https://react.dev/learn/start-a-new-react-project#can-i-use-react-without-a-framework) has more to say about using React without the use of a framework like Next or Remix.
```

Comment (kristersd) [reply src/features/commands.ts:1212]: I think the suggestions are great! Nice changes, Carl.

**Review by kristersd — APPROVED** [review]

**Pull request merged**